addons/website_sale/controllers/main.py

```diff
@@ -1379,6 +1383,9 @@ def confirm_order(self, **post):
     # ------------------------------------------------------
     # Extra step
     # ------------------------------------------------------
+    def _extra_info_values(self, **post):
```

Comment:
Same here. Also, `extra`, `info`, `values` is exactly the same as bazar machin truc. It's too generic. Explain the purpose of this method.

Reply:
In fact, `_extra_info` is a reference to the template rendered using the values returned by this function (the template's name: `website_sale.extra_info`). I tried to use the same convention as what was already used in website_sale (see for instance `checkout_values` called before rendering the `website_sale.checkout` template, or `_get_shop_payment_values` called before rendering the `website_sale.payment` template). 
Also, I added the same docstring as for `_extra_info_values`.